docs/user/FAQ.md

```diff
@@ -1,5 +1,55 @@
 # FAQ
 
-## Q: On Mac OS X, I neither can delete a node template nor a relationship template.
+## Q: What is TOSCA?
+
+A: Topology and Orchestration Specification for Cloud Applications (TOSCA) is an OASIS specification, which enables the modeling, provisioning and management of cloud applications. 
+For more detail, please click [here](http://eclipse.github.io/winery/tosca/).
```

Comment:
I would say `For more details see our [Notes on TOSCA](https://eclipse.github.io/winery/tosca/).`
Please always use `https`.